hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/util/OMMultipartUploadUtils.java

```diff
@@ -93,8 +91,7 @@ public static String getMultipartOpenKey(String volumeName,
        OMMetadataManager omMetadataManager, BucketLayout bucketLayout)
       throws IOException {
     if (bucketLayout == BucketLayout.FILE_SYSTEM_OPTIMIZED) {
-      return getMultipartOpenKeyFSO(volumeName, bucketName,
-          keyName, multipartUploadId, omMetadataManager);
+      return omMetadataManager.getMultipartKeyFSO(volumeName, bucketName, keyName, multipartUploadId);
     } else {
       return getMultipartOpenKey(volumeName, bucketName,
```

Comment:
I believe we can remove this method as well since there no other usages for the method.

Reply:
```suggestion
      return omMetadataManager.getMultipartKey(volumeName, bucketName,
```

Reply:
Thanks, fixed.